thoth/user_api/api_v1.py

```diff
@@ -397,9 +397,9 @@ def _get_log(node_name: str, analysis_id: str, namespace: str) -> typing.Tuple[t
         return result, 200
 
 
-def get_advise_python_status(analysis_id: str):
+def get_advise_python_status(analysis_id: str) -> typing.Tuple[typing.Dict[str, typing.Any], int]:
     """Get status of an adviser run."""
-    return _get_workflow_status(locals(), "adviser-", Configuration.THOTH_BACKEND_NAMESPACE)
+    return _get_status("adviser-", analysis_id, namespace=Configuration.THOTH_BACKEND_NAMESPACE)
```

Comment:
should it be `"advise"` instead of `"adviser-"`?

Reply:
Try these locally. I think it should be `advise` as it corresponds to the task in the workflow. See thoth-common logic you implemented earlier.

Reply:
it's `advise`

Reply:
You can read others from the manifest files: https://github.com/thoth-station/thoth-application